pulp_ansible/app/tasks/collections.py

```diff
@@ -448,6 +453,49 @@ async def _fetch_collection_metadata(self, requirements_entry):
 
         await asyncio.gather(*tasks)
 
+    async def _fetch_unpaginated_collection_metadata(self, name, namespace, requirement):
```

Comment:
```suggestion
    async def _read_from_downloaded_metadata(self, name, namespace, requirement):
```

Reply:
I'm proposing this because no actual fetching (aka downloading) is being done here.